src/js_identifiers.rs

```diff
@@ -1,12 +1,12 @@
-use unicode_id::UnicodeID;
+use unicode_id_start;
```

Comment:
the lint check is right, you are using fully qualified calls down below, so no need for this here.